lib/acts_as_list/active_record/acts/sequential_updates_method_definer.rb

```diff
@@ -4,7 +4,12 @@ def self.call(caller_class, column, sequential_updates_option)
       define_method :sequential_updates? do
         if !defined?(@sequential_updates)
           if sequential_updates_option.nil?
-            table_exists = caller_class.connection.table_exists?(caller_class.table_name)
+            table_exists =
+              if defined?(caller_class.connection.data_source_exists?)
```

Comment:
I think I'd rather be more explicit about why we're forking in the code here. Can we use `if ActiveRecord::VERSION::MAJOR >= 5` instead? Then we know we can remove this in the future if we drop Rails < 5 support.